### ISSUE TITLE
pCCDS Overlap Derivatives

### ISSUE THREAD

**Issue by carlosevmoura** [theochem/PyCI#42]
To proceed with implementing FanPT capabilities in PyCI, we need to calculate the derivatives of the overlap matrices.
This feature is already implemented for AP1roG, but is not currently available in the code for pCCD+S.

**Comment by msricher** [comment]
Yes, we had some difficulties with this. It's not high priority right now. Jacobian functions for the open-shell FanCI ansatze are very difficult.  Is it possible to use a numerical approximation for now?
